docs/common/community/blogs.md

```diff
@@ -1,6 +1,5 @@
 ---
 url: blogs
-weight: 200
```

Comment:
Thanks for catching this! These were left over from when these pages were moved from `versioned` to `common` and you're right, they're no longer needed here.